pubtools/_quay/operator_pusher.py

```diff
@@ -420,30 +429,64 @@ def build_index_images(self):
         iib_results = {}
 
         for version, items in sorted(self.version_items_mapping.items()):
-            bundles = [self.public_bundle_ref(i) for i in items]
-            all_archs = [
-                i.metadata["arch"] if i.metadata["arch"] != "x86_64" else "amd64" for i in items
-            ]
-            archs = sorted(list(set(all_archs)))
-            signing_keys = sorted(list(set([item.claims_signing_key for item in items])))
+            is_hotfix = any([item.metadata.get("com.redhat.hotfix") for item in items])
```

Comment:
Is it possible that only some push items will contain this?

Reply:
@querti : Can you please elaborate more on your question?
FWIW, when it comes to advisories created in errata, there would be at least one  docker image that will have "com.redhat.hotfix" label and there could be other related images and builds in the same advisory that don't have the label necessarily. You can find more info in https://issues.redhat.com/browse/CWFENGINE-585.

Reply:
I think my point was, we assume that every item is hotfix if only some items (not all) contain com.redhat.hotfix. Is this correct behavior?

Reply:
yes, it's not possible to mix hotfix items with non hotfix items in one push